qiskit/qobj/pulse_qobj.py

```diff
@@ -249,7 +249,12 @@ def __init__(self, meas_level, meas_return, pulse_library,
                 measurement driver LO's in GHz.
             memory_slot_size (int): Size of each memory slot if the output is
                 Level 0.
-            rep_time (int): Repetition time of the experiment in μs
+            rep_time: Time per program execution in sec. Must be from the list provided
+                by the backend (``backend.configuration().rep_times``).
+            rep_delay: Delay between programs in sec. Only supported on certain
+                backends (``backend.configuration()['dynamic_reprate_enabled']`` ).
```

Comment:
```suggestion
                backends (``backend.configuration().dynamic_reprate_enabled``).
```

Reply:
Addressed in https://github.com/Qiskit/qiskit-terra/pull/4539/commits/bcfd9c4d65c352c231fb033a4239dfdf9f8a2bc4.